pkgs/checks/example/example.dart

```diff
@@ -8,16 +8,16 @@ import 'package:test/scaffolding.dart';
 void main() {
   test('sample test', () {
     final someValue = 5;
-    checkThat(someValue).equals(5);
+    someValue.must.equal(5);
```

Comment:
So we rename the package to `pkg:must`? 😄 

`pkg:should` is nice...

Reply:
It feels a bit weird to have "values" floating around not wrapped in something.

Reply:
I do agree it feels weird to have this be a getter. I could probably get used to it though.